apm-agent-api/src/main/java/co/elastic/apm/api/CaptureSpan.java

```diff
@@ -87,4 +87,16 @@
      * </p>
      */
     boolean discardable() default true;
+
+    /**
+     * <p>
+     * Makes the captured span an exit span. Exit spans are usually meant to represent outgoing calls to external systems, like databases.
+     * When an exit span is created no nested span is created. For example when creating exit spans
+     * for a database, it can hide the fact that calls to this database are made through an HTTP client.
+     * </p>
+     * <p>
+     * When set to {@literal true}, it is recommended to also set values for {@link #type()} and {@link #subtype()}
+     * </p>
+     */
+    boolean exit() default false;
```

Comment:
I think this should be something like `asExit`. `discardable` is something that is clearly true or false, but `exit` is a command to do something rather than a boolean state

Reply:
or maybe `makeExit`

Reply:
Let's pick `asExit`, I've fixed that in latest commit.